docs/Monitoring.md

```diff
@@ -17,6 +17,7 @@ Add Helm 3 [Bitnami](https://hub.helm.sh/charts/bitnami) repository
 Install Prometheus Operator using Helm3
 
     cd ./charts/prometheus-operator
+    kubectl create secret generic alertmanager-prometheus-prometheus-oper-alertmanager --from-file=alertmanager.yaml
```

Comment:
Can you add this to the Makefile please?

Reply:
Sure. I will add it to one-click install command,
but now I have some difficulties in making appropriate configs work.